reference/calicoctl/ipam/show.md

```diff
@@ -21,6 +21,8 @@ Options:
   -h --help             Show this screen.
      --ip=<IP>          Report whether this specific IP address is in use.
      --show-blocks      Show detailed information for IP blocks as well as pools.
+     --show-borrowed    Show detailed information for "borrowed" IP addresses.
+     --configuration    Show current Calico IPAM configuration
```

Comment:
Hm, didn't spot this in code review - should probably be `--show-configuration` to be symmetric with the other options. 

Should be a quick fix.